apps/mylar3/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+#shellcheck disable=SC1091
+test -f "/scripts/umask.sh" && source "/scripts/umask.sh"
+test -f "/scripts/vpn.sh" && source "/scripts/vpn.sh"
+
+
+# Create the config dir if we need too
+mkdir -p /config
+
+#shellcheck disable=SC2086
+exec \
+    /usr/bin/python3 \
+        /app/Mylar.py \
+        --nolaunch \
+        --port ${MYLAR3__PORT:-8090} \
```

Comment:
Since you already set the default in the Dockerfile, no need to set a default here as well. I think I was double defining things in some containers and that should probably be cleaned up by me as well.

```suggestion
        --port ${MYLAR3__PORT} \
```

Reply:
👍